ClinGen/clincoded
# Connects to #450. Demo site styles

* Change styling on demo notice bar
* Make notice bar unclosable
* Add background to demo site

Testing:
* Make sure above changes are visible on local and test instances
* Make sure above changes are *not* visible on http://curation.clinicalgenome.org/ (use hosts file trick outlined in #353)

![image](https://cloud.githubusercontent.com/assets/4326866/11104400/58ec5dde-887d-11e5-8e53-f32cb221b3f6.png)
